src/components/Button/Button.js

```diff
@@ -97,6 +85,44 @@ Button.propTypes = {
    * adds optional value tag to the button
    */
   value: PropTypes.string,
+  /**
+   * Button flavor: string or strings in array. Flavors: neutral,
+   brand, destructive, icon, icon-border-filled, icon-container, icon-inverse,
+   icon-border, icon-bare, icon-x-small, reset, success, x-small, small, large
+   */
+  flavor: PropTypes.oneOfType([PropTypes.oneOf([
```

Comment:
These PropTypes are really verbose, please use a util that outputs them. E.g. `flavorPropType([arrayOfFlavors])`